operator-common/src/main/java/io/strimzi/operator/common/operator/resource/StatusUtils.java

```diff
@@ -29,18 +29,22 @@ public static String iso8601Now() {
     }
 
     public static Condition buildConditionFromException(Throwable error) {
+        return buildCondition(error == null ? "Ready" : "NotReady", error);
+    }
+
+    public static Condition buildCondition(String type, Throwable error) {
         Condition readyCondition;
         if (error == null) {
             readyCondition = new ConditionBuilder()
                     .withLastTransitionTime(iso8601Now())
-                    .withType("Ready")
-                    .withStatus("True")
+                    .withType(type)
+                    .withStatus("Ready")
```

Comment:
Did we mean to change the status here?

Reply:
You are absolutely right. It should be just "True" as it is today.